.changeset/empty-poems-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Updated `SubNav` links and submenus to use `ul` and `li` elements — instead of `div` elements — to communicate the hierarchy of the navigation to assistive technologies. All top-level items are all rendered as `li` elements, and submenus are rendered as `ul` elements with `li` children.
```

Comment:
Perfect 👌

Reply:
Nit, but could just go with the first line, as it conveys the sentiment and scope of the change clearly enough..

```suggestion
Updated `SubNav` links and submenus to use `ul` and `li` elements — instead of `div` elements — to communicate the hierarchy of the navigation to assistive technologies.
```